packages/interface-connection-manager/src/index.ts

```diff
@@ -6,7 +6,7 @@ import type { Multiaddr } from '@multiformats/multiaddr'
 
 export interface ConnectionManagerEvents {
   /**
-   * This event will be triggered anytime a new Connection is established to another peer.
+   * This event will be triggered any time a new Connection is established to another peer.
```

Comment:
Apparently [anytime](https://www.dictionary.com/e/anytime-vs-any-time/) is an adverb 😉